scripts/zones/Inner_Horutoto_Ruins/IDs.lua

```diff
@@ -45,6 +45,14 @@ zones[xi.zone.INNER_HORUTOTO_RUINS] =
             [17563799] = 17563801, -- -237.426 0.5 -23.412
             [17563800] = 17563801, -- -230.732 -0.025 -52.324
         },
+        MAGICKED_BONES_PH_DAGGER =
+        {
+            [17563669] = 17563671,
+        },
+        MAGICKED_BONES_PH_CLUB =
+        {
+            [17563668] = 17563670,
+        },
```

Comment:
```lua
MAGICKED_BONES_PH_CLUB = GetFirstID('Magicked_Bones')
MAGICKED_BONES_PH_DAGGER = MAGICKED_BONES_PH_CLUB +1
```

Reply:
or use the other get by name function that returns a table

Reply:
Doesn't this remove info though since the format seems to be [placeholder] = popped_mob_id?

Reply:
no it doesn't remove info. it gets the info in a more future proof manner that evades problems from ID shifts. you are copying an old pattern that's being slowly phased out.

Reply:
you really only even need one of these when the others ID is just +1 from it

Reply:
Ah I wasn't aware that was an old pattern, I was trying to make it fit the rest of the file. Does GetFirstID also limit it to the entries for the zone it is in, since there are Magicked bones in a bunch of other zones too?

Reply:
![image](https://github.com/LandSandBoat/server/assets/131182600/4842696f-0605-441e-876e-129777b8ecbc)

pulls only from the zone

Reply:
Ok cool